apps/server/src/modules/provisioning/strategy/oidc/service/oidc-provisioning.service.ts

```diff
@@ -185,4 +187,43 @@ export class OidcProvisioningService {
 
 		return filteredUsers;
 	}
+
+	async removeUserFromExternalGroups(
```

Comment:
In my opinion this name only describes the half of the functionality. Maybe you can find a better name :)